Document Globalnet GKE pre-requisites

sridhargaddam opened the pull request:
By default, GKE clusters do not allow services to be created with
ExternalIPs as it deploys the DenyServiceExternalIPs admission controller.
When deploying Globalnet on GKE clusters, the admission controller
has to be explicitly disabled. This PR captures this prerequisite.

Fixes: https://github.com/submariner-io/submariner-website/issues/742
Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

sridhargaddam committed:
Document Globalnet GKE pre-requisites

By default, GKE clusters do not allow services to be created with
ExternalIPs as it deploys the DenyServiceExternalIPs admission controller.
When deploying Globalnet on GKE clusters, the admission controller
has to be explicitly disabled. This PR captures this prerequisite.

Fixes: https://github.com/submariner-io/submariner-website/issues/742
Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

sridhargaddam commented:
@dfarrell07 the linter error seems to be unrelated to this PR.

dfarrell07 commented:
@sridhargaddam Yes it's broken because of the recent large git breakage, not this PR.

dfarrell07 committed:
Edit GKE/Globalnet preparation docs

The formatting objects like `ExternalIPs` is complex, but I think this
fits the K8s docs guidance and it is done similarly upstream:
https://kubernetes.io/docs/concepts/services-networking/service/#external-ips

Signed-off-by: Daniel Farrell <dfarrell@redhat.com>

dfarrell07 commented:
@sridhargaddam I sent a second commit with some edits, please let me know if it's still good with you. Thanks!

dfarrell07 commented:
This seems to address the user's original docs gap: https://app.slack.com/client/T09NY5SBT/C010RJV694M/thread/C010RJV694M-1650557876.652489
